src/client/app/actions/graph.js

```diff
@@ -37,23 +36,27 @@ export function updateBarDuration(barDuration) {
 export function changeBarDuration(barDuration) {
 	return (dispatch, state) => {
```

Comment:
I think this parameter should idiomatically be called `getState()`, not `state`, since it's a function that returns state instead of the actual state. This occurs in a few places.